binary/README.md

```diff
@@ -1,81 +1,89 @@
-<h1> Binary Toolset </h1>
+# Binary Toolset
 The binary toolset consists of a tool that translates the VSS YAML specification to the binary file format (see below),
 and two libraries that provides methods that are likely to be needed by a server that manages the VSS tree, one written in C, and one in Go.<br>
 
-<h3>Binary Parser Usage </h3>
+## Binary Parser Usage
 The translation tool can be invoked via the make file available on the VSS repo (https://github.com/COVESA/vehicle_signal_specification):
 
-```
-$ make binary
+```bash
+make binary
 ```
 or, by invoking all tools:
 
-```
-$ make all
+```bash
+make all
 ```
 
-To <b>run the binary tool without using the make file</b>, the binary tool library must first be built in the binary directory, then the vspec2binary.py is executed in the root directory:
+To **run the binary tool without using the make file**, the binary tool library must first be built in the
+binary directory, then the binary exporter of `vspec exporter` is executed in the root directory:
 
+```bash
+cd binary
+gcc -shared -o binarytool.so -fPIC binarytool.c
+cd <vss-root>
+vspec export binary -u spec/units.yaml --vspec spec/VehicleSignalSpecification.vspec -o vss.binary
```

Comment:
Now you need the `-b`argument as well?

Reply:
Better to just mention it as a limitation, there are some challenges including a binary in pip package, as it might need to be different for different architectues (amd64, arm64, riscv64, ...)

Reply:
Yes exactly. That ist why I thought to better make the dependency to the dll explicit and required